bucket/simple-dnscrypt.json

```diff
@@ -0,0 +1,63 @@
+{
+    "version": "0.7.1",
+    "description": "An easy to use GUI for dnscrypt-proxy",
+    "homepage": "https://simplednscrypt.org/",
+    "license": "MIT",
+    "suggest": {
+        "vcredist2015-2022": "extras/vcredist2022"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/bitbeans/SimpleDnsCrypt/releases/download/0.7.1/SimpleDNSCrypt64.msi",
+            "hash": "5BA0C94774ACD7ADE18446B4351E7D28A948C490AEFC50DC6C08C13551F19F7F"
+        },
+        "32bit": {
+            "url": "https://github.com/bitbeans/SimpleDnsCrypt/releases/download/0.7.1/SimpleDNSCrypt.msi",
+            "hash": "CAC6B8119AF3DA70A5977A2E73EE5214303990FEDD7ABBC5F247A1E2CF19A663"
+        }
+    },
+    "pre_install": [
+        "if (!(Test-Path \"$env:LOCALAPPDATA\\SimpleDnsCrypt\")) {New-Item -Path \"$env:LOCALAPPDATA\" -Name \"SimpleDnsCrypt\" -ItemType \"directory\" -Force | Out-Null}",
+        "New-Item \"$dir\\dnscrypt-proxy\\blacklist.txt\" -ItemType \"file\" | Out-Null",
+        "New-Item \"$dir\\dnscrypt-proxy\\blocked.log\" -ItemType \"file\" | Out-Null",
+        "New-Item \"$dir\\dnscrypt-proxy\\query.log\" -ItemType \"file\" | Out-Null",
+        "New-Item \"$dir\\dnscrypt-proxy\\domain-blacklist.txt\" -ItemType \"file\" | Out-Null",
+        "New-Item \"$dir\\dnscrypt-proxy\\domain-whitelist.txt\" -ItemType \"file\" | Out-Null"
+    ],
+    "shortcuts": [
+        [
+            "SimpleDnsCrypt.exe",
+            "Simple DnsCrypt"
+        ]
+    ],
+    "bin": "SimpleDnsCrypt.exe",
+    "persist": [
+        "dnscrypt-proxy\\dnscrypt-proxy.toml",
+        "dnscrypt-proxy\\blacklist.txt",
+        "dnscrypt-proxy\\blocked.log",
+        "dnscrypt-proxy\\query.log",
+        "dnscrypt-proxy\\domain-blacklist.txt",
+        "dnscrypt-proxy\\domain-whitelist.txt"
+    ],
+    "pre_uninstall": [
+        "if (!(is_admin)) { error \"$app requires admin rights to $cmd\"; break }",
+        "",
+        "Start-Process \"$dir\\Uninstall.exe\" | Out-Null",
+        "if (Get-Process -Name \"SimpleDnsCrypt\" | Out-Null) {Stop-Process -Name \"SimpleDnsCrypt\" | Out-Null}",
+        "if (Get-Service -Name \"dnscrypt-proxy\" | Out-Null) {Stop-Service -Name \"dnscrypt-proxy\"| Out-Null; Remove-Service -Name \"dnscrypt-proxy\" | Out-Null}"
```

Comment:
```suggestion
        "if (Get-Process -Name 'SimpleDnsCrypt' -ErrorAction SilentlyContinue) { Stop-Process -Name 'SimpleDnsCrypt' | Out-Null }",
        "$service = Get-Service -Name 'dnscrypt-proxy' -ErrorAction SilentlyContinue",
        "if ($service.Status -ne 'Stopped') {",
        "    if (!(is_admin)) { error 'Admin rights are required to stop dnscrypt-proxy service'; break }",
        "    Stop-Service -Name 'dnscrypt-proxy' | Out-Null",
        "}",
        "if (($service) -and ($cmd -eq 'uninstall')) {",
        "    if (!(is_admin)) { error 'Admin rights are required to remove dnscrypt-proxy service'; break }",
        "    Remove-Service -Name 'dnscrypt-proxy' | Out-Null",
        "}"
```

Reply:
Looks better than mine, I'll go with it.